src/pages/gift-certificate/gift-certificate.js

```diff
@@ -0,0 +1,105 @@
+import { Button, TextField } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { useFormik } from 'formik';
+import { INITIAL_CERTIFICATE_COUNT, MATERIAL_UI_COLOR, TEXT_FIELD_VARIANT } from '../../configs';
+import { useStyles } from './gift-certificate.styles';
+import { validationSchema } from '../../validators/email';
+import { certificateRules } from '../../locales/en/certificates.json';
+import CertificateCheckbox from './certificate-checkbox';
+
+const GiftCertificate = () => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+  const email = useSelector(({ User }) => User?.userData?.email);
+  const [initialValues, setInitialValues] = useState('');
+
+  const [checkboxesArr, setCheckboxesArr] = useState([
+    { value: 500, checked: false, count: INITIAL_CERTIFICATE_COUNT },
+    { value: 1000, checked: true, count: INITIAL_CERTIFICATE_COUNT },
+    { value: 1500, checked: false, count: INITIAL_CERTIFICATE_COUNT }
+  ]);
```

Comment:
maybe we should put this array into a separate constant and pass it here

Reply:
Resolved